beetsplug/discogs.py

```diff
@@ -56,7 +60,9 @@
     ConnectionError,
     socket.error,
     http.client.HTTPException,
+    HTTPError,
     ValueError,  # JSON decoding raises a ValueError.
+    JSONDecodeError,  # Except sometimes it raises a `JSONDecodeError`
```

Comment:
Unneeded - `JSONDecodeError` inherits from `ValueError` which is already present in the list.